python/eos/analysis.py

```diff
@@ -22,6 +22,9 @@
 import scipy
```

Comment:
Please adjust the commit message according to [the instuctions in the wiki](https://github.com/eos/eos/wiki/How-to-Contribute)